src/openafval/trash/models.py

```diff
@@ -0,0 +1,263 @@
+from __future__ import annotations
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class BagObject(models.Model):
+    id: int
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_("Imported identifier for the BAG object."),
+        max_length=255,
+        unique=True,
+    )
+    address = models.CharField(
+        _("address"),
+        help_text=_("The address of the BAG object."),
+        max_length=80,
+        blank=True,
+    )
+
+    class Meta:  # pyright: ignore
+        verbose_name = _("BAG object")
+        verbose_name_plural = _("BAG objects")
+
+    def __str__(self):
+        return self.address or self.identifier
+
+
+class Entity(models.Model):
```

Comment:
"Entity" is the least descriptive term there is (because everything is an "entity"). Perhaps "Owner" since this is what the model actually models: the owner of a BAG object. Or even "BagObjOwner" or something (what's best will depend on the relations and reverse relations we need to keep track of, see comment below on `EntityObjectManagement`.